### PR TITLE
Dark/Light mode for preloaded html

### PR DESCRIPTION
**Pull request by hungdoansy**
# Description
There's currently a flicker when users go the page. It happens because the preloaded HTML is only in Dark mode. This PR contains:
- Dark/Light mode for the preloaded html
- Check for dark/light mode right when the html is loaded

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -163,19 +192,61 @@
...
+          class="onlyVisibleInLightMode"
+          style="animation: pulse 800ms linear infinite; margin: auto;"
+          width="160px"
+          src="/logo.svg"
+          alt="DmmLogo"
+        />
       </div>
     </div>
+    <script>
+      ;(() => {
```
Comment (nguyenhoaidanh) [inline public/index.html:217]: `;` redundant

Comment (hungdoansy) [reply public/index.html:217]: It's auto added

Comment (hungdoansy) [reply public/index.html:217]: Maybe this is the answer: https://stackoverflow.com/questions/17978883/what-is-the-purpose-of-a-semicolon-before-an-iife

Comment (nguyenhoaidanh) [reply public/index.html:217]: cool

**Comment by nguyenhoaidanh** [comment]
(using dark mode)
- change to light mode => f5  => it shows dark mode

**Comment by hungdoansy** [comment]
> (using dark mode)
> 
> * change to light mode => f5  => it shows dark mode

I'm fixing it. But please wait 1-2s before reloading. It delays 300-400ms before saving data to localStorage

**Review by viet-nv — APPROVED** [review]

**Pull request merged**